startleft/api/controllers/health/health_controller.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import APIRouter
+from startleft.api.api_config import ApiConfig
+
+PREFIX = ''
+URL = '/health'
+RESPONSE_BODY = {"status": "StartLeft server is ok"}
+
+router = APIRouter(prefix=PREFIX)
+
+
+@router.get(URL, tags=["health"])
+async def health():
```

Comment:
Could it be useful to perform a request against the configured IriusRisk server to return if this instance has access to it?